sriovnet.go

```diff
@@ -254,9 +256,11 @@ func SetVfPrivileged(handle *PfNetdevHandle, vf *VfObj, privileged bool) error {
 	/* do not check for error status as older kernels doesn't
 	 * have support for it.
 	 */
-	netlink.LinkSetVfTrust(handle.pfLinkHandle, vf.Index, trusted)
-	netlink.LinkSetVfSpoofchk(handle.pfLinkHandle, vf.Index, spoofChk)
-	return nil
+	err = netlink.LinkSetVfTrust(handle.pfLinkHandle, vf.Index, trusted)
```

Comment:
@moshe010 This breaks support on older kernels. Please rework this part to continue using existing way.

Reply:
ok I revert the code to be like it was and put nolint to ignore the complains for now. I don't want this to block this PR.